### ISSUE TITLE
Alert annotation on station change

### ISSUE THREAD

**Issue by PatrickCleary** [transitmatters/t-performance-dash#801]
Steps to reproduce:
1. Go to trips page with alerts: (https://dashboard.transitmatters.org/red/trips/single/?to=70075&from=70067&date=2023-07-20)
2. Open alerts modal, select alert to apply to graph.
3. Switch `to` station param
4. Chart will keep annotation on headways, but not travel times.

![Image](https://user-images.githubusercontent.com/46229349/254922981-8151a8ca-25e1-4e71-8ae3-280100deddd8.png)

**Comment by skaplan-dev** [comment]
I don't see any yellow boxes on prod 🤔

**Comment by PatrickCleary** [comment]
Did you select an alert from the alerts modal? The modal comes up from tapping the warning emoji on the bottom right.

**Comment by skaplan-dev** [comment]
> Did you select an alert from the alerts modal? The modal comes up from tapping the warning emoji on the bottom right.

Ah I see, Honestly I think we should just add them to the graphs by default with a tooltip of what the delay was for

**Comment by rudiejd** [comment]
I looked into this a bit. I was able to reproduce locally. From the component state, it looks like the travel times chart is actually  the "correct" one in that the alerts aren't applied after you change one of the query parameters. The other components must just not be getting repainted for some reason. 
![image](https://github.com/transitmatters/t-performance-dash/assets/46619169/272df8b4-0d03-4f5d-832b-79837cce11f5)
